daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -168,12 +168,9 @@ object VersionTimeline {
   }
 
   private[lf] val stableLanguageVersions =
-    VersionRange(
-      min = LanguageVersion(Major.V1, Minor("6")),
-      max = LanguageVersion(Major.V1, Minor("8")),
-    )
+    VersionRange(min = LanguageVersion.v1_6, max = LanguageVersion.v1_8)
```

Comment:
Don't we have a constant for "maximum stable version"? Shouldn't we use that here as the upper bound instead?

Reply:
I will drop soon the `VersionTimeline` so we should not worry too much.
I will try to take this consideration when cleaning further `LanguageVersions`.

Reply:
Cool. Thanks.